md_to_bgg.py

```diff
@@ -164,7 +164,7 @@ def render_heading(self, element):
 
         # There is no concept of header in BGG markup, so we simulate this by
         # using the default Large and Huge font sizes:
-        size = {1: 24, 2: 18}[element.level]  # Maximum 2 levels!
+        size = {1: 24, 2: 18, 3: 14, 4: 12, 5: 11, 6: 10}[element.level]  # All 6 levels!
```

Comment:
The last 2–3 levels do not really work because they are smaller than (or identical to) the default text on BoardGameGeek (which appears to be of size 12).

This can be seen with the following BGG Markup:
```
[size=14]14 size[/size]
Normal text

[size=12]12 size[/size]
Normal text

[size=10]10 size[/size]
Normal text
```

Supporting 6 levels would bring `md_to_bgg` closer to CommonMark, so it doesn't hurt to support them.

We just need to go from size 24 to size, say, 14 in 6 steps. This doesn't leave much liberty, leading for example to something like sizes 24, 22… 16, 14. The problem with this is that the most commonly used headers (level 1–2 or maybe 1–3) are not so easy to distinguish anymore, which is not so nice for users.

We could have levels 4–6 be in size 12 (normal text), maybe, with bold and italics used for distinguishing between them? What do you think?